content/en/releases/_index.md

```diff
@@ -2,6 +2,7 @@
 linktitle: Release History
 title: Releases
 type: docs
+layout: release-info
```

Comment:
Should every page within `/releases` have this marker? Maybe it's not needed, as we have `if eq .Section "releases"` logic elsewhere.